<?xml version="1.0" encoding="utf-8" standalone="yes"?><rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom"><channel><title>Tags/Dynamodb on Cino</title><link>https://cino.io/tags/dynamodb/</link><description>Recent content in Tags/Dynamodb on Cino</description><generator>Hugo</generator><language>en-us</language><lastBuildDate>Sat, 22 Mar 2025 12:00:40 +0100</lastBuildDate><atom:link href="https://cino.io/tags/dynamodb/index.xml" rel="self" type="application/rss+xml"/><item><title>Modifying DynamoDB TTL with CDK</title><link>https://cino.io/2025/modifying-dynamodb-ttl-with-cdk/</link><pubDate>Sat, 22 Mar 2025 12:00:40 +0100</pubDate><guid>https://cino.io/2025/modifying-dynamodb-ttl-with-cdk/</guid><description>&lt;p>Ever tried to update the TTL attribute of a DynamoDB table using the AWS CDK and got a &lt;code>InvalidRequest&lt;/code> in CDK or a &lt;code>ValidationException&lt;/code> via the CLI? I did, and it took me a while to figure out why. In this post, I&amp;rsquo;ll explain what happened and how to avoid the same issue in the future. This is a short post, but I think it&amp;rsquo;s worth sharing because it can save you some time and frustration.&lt;/p></description></item></channel></rss>